taxcalc/utils.py

```diff
@@ -2,13 +2,12 @@
 import pandas as pd
 from pandas import DataFrame
 
-STATS_COLUMNS = ['c00100', 'c04100', 'c04470', 'c04800', 'c05200',
-                 'c09600', 'c07100', 'c09200', '_refund', '_ospctax',
-                 'c10300', 'e00100', 's006']
+STATS_COLUMNS = ['c00100', '_standard', 'c04470', 'c04600','c04800', 'c05200',
+                 'c09600', 'c05800', 'c09200', '_refund', 'c07100','_ospctax',
+                 's006']
 
-TABLE_COLUMNS = ['c00100', 'c04100', 'c04470', 'c04800', 'c05200',
-                 'c09600', 'c07100', 'c09200', '_refund', '_ospctax',
-                 'c10300']
+TABLE_COLUMNS = ['c00100', 'num_returns_StandardDed','_standard', 'num_returns_ItemDed', 'c04470',
```

Comment:
The value of `TABLE_COLUMNS` was based on the understanding I had for what was needed for the web app. Are these the new columns needed for display for the webapp? If so, there is no issue. If not (and therefore the original value of `TABLE_COLUMNS` is needed), a separate variable called `WEB_APP_TABLE_COLUMNS` or similar should be created.